api/RTCStatsReport.json

```
@@ -5359,7 +5359,7 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "≤73"
+                "version_added": "≤60"
```

See: https://bugzilla.mozilla.org/show_bug.cgi?id=1348657

```suggestion
                "version_added": "55"
```